Bottom: e045b931472d6054c0236a8dc2b6936d1e04270b
Top:    0ae11f2f8cfd858a36140403b84ba73ebc0b220a
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-02-07 14:37:30 +0530

Refresh of sched-fair-paraphrase

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 6c0cb349a02f..492d327baf92 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5458,7 +5458,7 @@ enum idle_cpu_type {
 	cpu_preempted_idle,
 	cpu_non_preempted_idle,
 	cpu_sched_idle
-}
+};
 
 /*
  * is_idle_cpu - is a given CPU idle for enqueuing work.
